source/enclave.haml

```diff
@@ -1,9 +1,7 @@
 ---
-title: Enclave - Secure DevOps
-description: Aptible Enclave is a container platform that simplifies secure
-             deployment and development best practices. Unlike using IaaS
-             directly, Enclave is turnkey for regulated apps, and includes
-             robust DevOps and security automation.
+title:  Container Orchestration Platform with Built-in Security Controls for Complying with HIPAA, ISO 27001, SOC 2, NIST 800-53, and many others - Aptible Enclave&reg;
```

Comment:
@henryhund have you verified that the `&reg;` glyph is rendered as expected in the title?